wagtail/admin/templates/wagtailadmin/shared/page_status_tag.html

```diff
@@ -1,18 +1,15 @@
-{% load i18n %}
-{% load wagtailadmin_tags %}
+{% load i18n wagtailadmin_tags wagtailcore_tags %}
 
 {% if page.live %}
-    {% with page_url=page.url %}
-        {% if page_url is not None %}
-            <a href="{{ page_url }}" target="_blank" rel="noreferrer" class="status-tag primary" title="{% trans 'Visit the live page' %}">
-                <span class="visuallyhidden">{% trans "Current page status:" %}</span> {{ page.status_string }}
-            </a>
-        {% else %}
-            {% status classname="primary" %}
-                <span class="visuallyhidden">{% trans "Current page status:" %}</span> {{ page.status_string }}
-            {% endstatus %}
-        {% endif %}
-    {% endwith %}
+    {% if page.url is not None %}
```

Comment:
The comment about using `pageurl` instead of `page.url` is defeated by this if

How about something like:

```jinja2
{% pageurl page as the_page_url %}
{% if the_page_url is not None %}
   <a href="{{ the_page_url }} ...
{% else %}
 ...
{% endif %}
```

? (note: did not test locally, but should work)

Reply:
Ahh yes I didn't realize I used already used, still working on it would get everything in prefer